config/rootfs/debos/scripts/buster-libcamera.sh

```diff
@@ -41,9 +41,15 @@ echo '{  "tests_suites": [' >> $BUILDFILE
 ########################################################################
 
 LIBCAMERA_URL=https://git.linuxtv.org/libcamera.git
+COMMIT_ID=0d50a04cc918e5122d1b22201cc949c2cfd337a5
 mkdir -p /var/tests/libcamera && cd /var/tests/libcamera
 
-git clone --depth=1 $LIBCAMERA_URL .
+git clone $LIBCAMERA_URL .
+
+git config --global user.email "bot@kernelci.org"
```

Comment:
It's worth adding a comment here to explain why reverting this commit is needed.

Reply:
done